traffic_detector.py

```diff
@@ -7,8 +7,8 @@ def car_detection():
     """
     Method to detect car objects from the traffic video
     """
-    car_classification = cv2.CascadeClassifier("C:\\Users\\bsens\\Downloads\\haarcascade_car.xml")
-    capt = cv2.VideoCapture("C:\\Users\\bsens\\Downloads\\MVI_6838.mp4")
+    car_classification = cv2.CascadeClassifier("haarcascade_car.xml")
+    capt = cv2.VideoCapture("MVI_6838.mp4")
```

Comment:
I guess this sort of works but I still think a command line parameter (or interactively asking for the filename) would be much better.